components/03-modules/cart-list-item/CartListItem.vue

```diff
@@ -0,0 +1,237 @@
+<template>
+  <div class="cart-list-item">
+    <div class="cart-list-item__image">
+      <alpaca-image
+        :src="productImage.dataSrc"
+        :alt="productImage.alt"
+      />
+    </div>
+    <div class="cart-list-item__info">
+      <div class="cart-list-item__data">
+        <alpaca-link
+          :href="href"
+          custom-class="link--inverted cart-list-item__name"
+        >
+          {{ productName }}
+        </alpaca-link>
+        <div class="cart-list-item__details">
+          <div class="cart-list-item__qty">
+            <alpaca-input
+              :id="id"
+              :label="qtyInput.label.text"
+              :placeholder="qtyInput.field.placeholder"
+              :name="name"
```

Comment:
More important than defining this as prop will be to have option to get value of this input and event when was changed